pkg/api/k8s_versions.go

```diff
@@ -48,6 +48,41 @@ var k8sComponentVersions = map[string]map[string]string{
 		"gchighthreshold":                  strconv.Itoa(DefaultKubernetesGCHighThreshold),
 		"gclowthreshold":                   strconv.Itoa(DefaultKubernetesGCLowThreshold),
 	},
+	"1.16": {
+		"dockerEngine":                     "1.13.*",
+		"dashboard":                        "kubernetes-dashboard-amd64:v1.10.1",
+		"exechealthz":                      "exechealthz-amd64:1.2",
+		"addon-resizer":                    "addon-resizer:1.8.5",
+		"heapster":                         "heapster-amd64:v1.5.4",
+		"metrics-server":                   "metrics-server-amd64:v0.3.5",
```

Comment:
Shouldn't this new metrics-server be in the 1.17 map?

Reply:
Nice catch–I'll fix it.